go.mod

```diff
@@ -9,12 +9,12 @@ require (
 	github.com/golang/protobuf v1.4.2
 	github.com/google/go-cmp v0.5.0
 	github.com/google/gofuzz v1.0.0 // indirect
+	github.com/grpc-ecosystem/grpc-gateway v1.14.6
 	github.com/kr/pretty v0.1.0 // indirect
+	github.com/open-telemetry/opentelemetry-proto v0.4.0
```

Comment:
This seems odd. Shouldn't the forced vendoring we are doing here alleviate the dependency on the Go code contained in `github.com/open-telemetry/opentelemetry-proto`?

Reply:
Yes, this is a result of running `make precommit` while having the git submodule directory populated.